runtime-common/stdlib/msgpack/object_visitor.h

```diff
@@ -65,10 +67,19 @@ class object_visitor : vk::not_copyable {
   void parse_error(size_t /*parsed_offset*/, size_t /*error_offset*/) const;
   void insufficient_bytes(size_t /*parsed_offset*/, size_t /*error_offset*/) const;
 
+  bool has_error() const {
+    return error.has_value();
+  }
+
+  std::optional<vk::string_view> get_error() const {
```

Comment:
Maybe `std::string_view`? It is available from c++17 and old runtime require this standart

Reply:
It doesn't have constructor from two pointers, that is why we have a wrapper for that and use it in both compiler and old runtime.